### ISSUE TITLE
Add more variance between focused/unfocused windows in default look.

### ISSUE THREAD

**Issue by kgilmer** [regolith-linux/regolith-desktop#231]
I common issues people find is the subtle UI treatment for unfocused windows.  By default it should be a little more obvious which window is in focus.  

Related: https://github.com/regolith-linux/regolith-desktop/issues/210

**Comment by cheginit** [comment]
How about using [`flashfocus`](https://github.com/fennerm/flashfocus)?

**Comment by kgilmer** [comment]
For now I'm going to just add a bit of contrast to the child border color of the focused window.  `flashfocus` looks cool but I'd like to get a change out sooner as it seems like this is one of the most common complaints new users have w/ Regolith.

If you'd like to test out `flashfocus` with Regolith I'd love to see what you come up with @cheginit :smile:

**Comment by cheginit** [comment]
So here are the steps for installing `flashfocus`:
```bash
sudo apt-get install libxcb-render0-dev libffi-dev python3-dev python3-cffi python3-pip
pip3 install --user flashfocus
```

Then this line needs to be added to `compton` config file:
```bash
opacity-rule = [
  "0:_NET_WM_STATE@:32a *= '_NET_WM_STATE_HIDDEN'"
];
```
Next up is the `i3` config file:
```bash
exec_always --no-startup-id flashfocus
bindsym $mod+Shift+n exec --no-startup-id nc_flash_window
```
Then copy my `flashfocus` config file:
```bash
mkdir ~/.config/flashfocus
wget -O ~/.config/flashfocus/flashfocus.yml https://raw.githubusercontent.com/cheginit/dotfiles/master/.config/flashfocus/flashfocus.yml
```

Now, whenever you change window you'd see a flash. By hitting mod+shift+n the current window flashes. Also, Firefox and a lone window in a workspace will not flash.

**Comment by kgilmer** [comment]
Fixed in 1.3.1.
